api/get.js

```diff
@@ -126,8 +139,13 @@ Api.prototype = {
         var lookup = {
             search: '?' + query,
             query: query,
-            path: req.url
+            binding: req.url
         };
+
+        // Append the query string to the base url
+        url += lookup.search;
```

Comment:
This is not needed. The goal of lookup is to be used in the config file. If a search string is needed then a {{search}} is added to the url param of the config file